CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.6.1
```

Comment:
```suggestion
## 3.7.0
```

The version also needs to change in the pubspec.

Reply:
Yes, also `lated` is weird, can we either call it `late` (preferred) or `isLate`?